optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -319,8 +346,11 @@ def __init__(
         sample_size=None,  # type: Optional[int]
         best_params=None,  # type: Optional[Dict[str, Any]]
         tuning_history=None,  # type: Optional[List[Dict[str, Any]]]
```

Comment:
Do you think it's too much to in this PR raise a deprecation warning if this argument is passed, recommending the usage of the new `study` parameter?

Reply:
I also deprecated the `tuning_history` argument in https://github.com/optuna/optuna/pull/1032/commits/894a666c774e2b27f1811efad1e853c6add77cb3. Thanks!